### ISSUE TITLE
eslint-plugin-prettier has a peerDependency on a really old prettier version

### ISSUE THREAD

**Issue by arobinson** [prettier/eslint-plugin-prettier#418]
eslint-plugin-prettier has a peer dependency of `"prettier": ">=1.13.0"`:

```
npm WARN eslint-plugin-prettier@3.4.0 requires a peer of prettier@>=1.13.0 but none is installed. You must install peer dependencies yourself.
```

In npm this means that a 1.x version is required at 1.13.0 or higher 1.x version.
Perhaps this is what it should be:?
```
  "prettier": ">=1.13.0 || 2"
```

This should be allowing prettier 2 versions as well.  

**What version of `eslint` are you using?**
^7.24.0

**What version of `prettier` are you using?**
2.2.1

**What version of `eslint-plugin-prettier` are you using?**
3.4.0

**Please paste any applicable config files that you're using (e.g. `.prettierrc` or `.eslintrc` files)**

**What source code are you linting?**
Typescript

**What did you expect to happen?**
No missing peer dependencies

**What actually happened?**
eslint-plugin-prettier was prettier version 1 installed

**Comment by BPScott** [comment]
`>=1.13.0` means any version after 1.13.0 including major versions. It already includes 2.*. You can see this using the [semer visualiser](https://semver.npmjs.com/)

**Comment by arobinson** [comment]
If that is the case, why does not issue the warning?

**Comment by BPScott** [comment]
Without a copy of your package.json  / lock file i wouldn't be able to speculate. Perhaps you've got duplicates in your tree?

`yarn why prettier` and `yarn why eslint-plugin-prettier` would also be useful diagnostic information
